src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__img"
+        src="./images/imac.jpeg"
+        alt="iMac 27 Retina 5K photo">
+      <h3 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="product-card__product-code">
+        Product code: 195434
+      </p>
```

Comment:
```suggestion
      <img
        class="product-card__img"
        src="./images/imac.jpeg"
        alt="iMac 27 Retina 5K photo"
       >
       
      <h3 class="product-card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h3>
      
      <p class="product-card__product-code">
        Product code: 195434
      </p>
```
Add one empty line between sibling blocks inside the same parent block for both of them.